warrant/__init__.py

```diff
@@ -346,6 +346,18 @@ def confirm_sign_up(self,confirmation_code,username=None):
         self._add_secret_hash(params, 'SecretHash')
         self.client.confirm_sign_up(**params)
 
+    def resend_confirmation_code(self,username=None):
+        """
+        Resend the confirmation for registration
+        :param username: User's username
+        """
+        if not username:
+            username = self.username
+        params = {'ClientId': self.client_id,
+                  'Username': username}
```

Comment:
You can remove 2LOC, but it's up to you.
```python
'Username': self.username if username is None else username
```

Reply:
Yeah of course, thanks for pointing out! Updated.